goosebit/api/v1/devices/device/routes.py

```diff
@@ -5,23 +5,22 @@
 
 from goosebit.api.v1.devices.device.responses import DeviceLogResponse, DeviceResponse
 from goosebit.auth import validate_user_permissions
-from goosebit.permissions import Permissions
 from goosebit.updater.manager import UpdateManager, get_update_manager
 
 router = APIRouter(prefix="/{dev_id}")
 
 
 @router.get(
     "",
-    dependencies=[Security(validate_user_permissions, scopes=[Permissions.HOME.READ])],
+    dependencies=[Security(validate_user_permissions, scopes=["home.read"])],
```

Comment:
Bit sad to switch from a "type-safe" approach to free form strings that can have typos. But ok for me.

Reply:
Yeah it's not my favorite either, but I think this makes more sense for allowing plugins.

I could try to make this more type safe, maybe by having the nav register permissions, then checking if all user permissions match a valid pattern, but that could be a problem with plugins.